core/src/integration-test/java/com/scalar/db/storage/cassandra/CassandraAdminTestUtils.java

```diff
@@ -0,0 +1,26 @@
+package com.scalar.db.storage.cassandra;
+
+import com.scalar.db.util.AdminTestUtils;
+import java.util.Properties;
+
+public class CassandraAdminTestUtils extends AdminTestUtils {
+
+  public CassandraAdminTestUtils(Properties properties) {
+    super(properties);
+  }
+
+  @Override
+  public void dropMetadataTable() {
+    // Do nothing
```

Comment:
Just a question.
I thought the metadata table is now managed for Cassandra as well for consistency, is it OK with empty?

Reply:
Thank you for taking a look! 

The change is not yet merged to the `master` branch. In the `feat/namespaces_management` branch, the metadata table is managed for Cassandra.

Reply:
Maybe there is some confusion but the `feat/namespaces_management` only implements the namespace management for Cassandra. 
Managing the table metadata using a metadata table for Cassandra is not yet implemented.

Reply:
Thank you for clarification!
So, I understand `Do nothing` parts will eventually be implemented after the table metadata for Cassandra is implemented.

Reply:
Yes, exactly.